### PR TITLE
feat(palette-mobile): move button

### PR DESCRIPTION
**Pull request by pvinis**
This PR is part of [MOPLAT-610] 

### PR Checklist

- [x] I have tested my changes on **iOS** and **Android**.
- [x] I hid my changes behind a **[feature flag]**, or they don't need one.
- [x] I have included **screenshots** or **videos**, or I have not changed the UI.
- [x] I have added **tests**, or my changes don't require any.
- [x] I added an **[app state migration]**, or my changes do not require one.
- [x] I have documented any **follow-up work** that this PR will require, or it does not require any.
- [x] I have added a **changelog entry** below, or my changes do not require one.

### To the reviewers 👀

- [ ] I would like **at least one** of the reviewers to **run** this PR on the simulator or device.

<details><summary>Changelog updates</summary>

#### Cross-platform user-facing changes

-

#### iOS user-facing changes

-

#### Android user-facing changes

-

#### Dev changes

- palette button from palette-mobile

</details>

Need help with something? Have a look at our [docs], or get in touch with us.

[app state migration]: ../blob/main/docs/adding_state_migrations.md
[feature flag]: ../blob/main/docs/developing_a_feature.md
[docs]: ../blob/main/docs/README.md

[MOPLAT-610]: https://artsyproduct.atlassian.net/browse/MOPLAT-610?atlOrigin=eyJpIjoiNWRkNTljNzYxNjVmNDY3MDlhMDU5Y2ZhYzA5YTRkZjUiLCJwIjoiZ2l0aHViLWNvbS1KU1cifQ

### CODE REVIEW

**Review by damassi — APPROVED** [review]

File: package.json [hunk 1]
```diff
@@ -176,7 +177,7 @@
     "react-native-localize": "2.0.1",
     "react-native-pager-view": "6.0.0",
     "react-native-push-notification": "8.1.1",
-    "react-native-reanimated": "2.13",
+    "react-native-reanimated": "2.14.4",
```
Comment (damassi) [inline package.json:180]: I tried bumping this to 14 at some point when working on the debugging, but no go (I think due to android not having the right image for RN 0.66). Are we sure this is ok? Let me see if i can dig up the related PR and if i made a note.

Comment (damassi) [reply package.json:180]: Ah yes 😢 https://github.com/artsy/eigen/pull/8003. See description

**Review by damassi — APPROVED** [review]
Merge when 🍏 (and assuming it runs on Android!) We can deal with the fires should they arise.

**Comment by pvinis** [comment]
yup. tested on android and ios (i made a beta a few min ago), they both seem 💯

**Pull request merged**